packages/guui/components/Grid/index.js

```diff
@@ -44,7 +41,9 @@ const columns: GridSizes = {
     },
 };
 
-const breakpointMqs = {
+const breakpointMqs: {
+    [Breakpoint]: string,
```

Comment:
Out of interest, the Breakpoint type represents one of a group of string values here. This is great for explicitness. Does it add any type safety with Flow over just using `string`? I.e. is there any way to leverage the additional typing we've added here?

Reply:
It prevents the keys of this object from being anything other than one of the `Breakpoint` strings

```js
// good
const breakpointMqs = {
  tablet: '@media(min-width: 740px)'
}

// bad
const breakpointMqs = {
  foo: '@media(min-width: 740px)'
}
```

I'm not sure if there's any real type safety advantage to it? Maybe  like you say, it's good for explicitness.

Reply:
Ah yep - no that makes sense. Thanks.